removed-features.asc

```diff
@@ -0,0 +1,77 @@
+:docinfo1:
+:experimental:
+
+= Removed Features
+:sectanchors:
+
+== Automatic night mode color inversion
+
+NOTE: Removed in AnkiDroid 2.17
+
+=== Feature Description
+
+AnkiDroid contained a very basic color inverter that e.g. changes white to black and black to white when night mode is enabled.
+This existed
```

Comment:
Just trying to avoid the use of contractions (`e.g.`) in docs to help folks using automated tranlators
Also trying to give a little bit of context on the controversy / explain *why* this would even matter (green to pink? ouch, that's the real invert though)

```suggestion
Previously, AnkiDroid contained a very basic color inverter for card content in night mode. For example it changed white to black and black to white. It inverted *all* colors though for example green inverts to pink. These changes were frequently unwanted.
```